x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_status_tags.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { EuiBadge, EuiText } from '@elastic/eui';
+import { UptimeSettingsContext } from '../../../contexts';
+
+const TextStyle = styled.div`
+  text-transform: uppercase;
+  font-weight: 600;
+`;
+
+const BadgeItem = styled.div`
+  margin-bottom: 5px;
+`;
+
+const TagContainer = styled.div`
+  padding: 10px;
+  max-height: 300px;
+  overflow: hidden;
+`;
+
+interface Props {
+  monitorLocations: any;
```

Comment:
Can we declare this prop as `monitorLocations: any[]` (or just name it `locations`) and pass `monitorLocations?.locations ?? []` from the parent?  That way we can avoid the nesting below and we won't be calling `forEach` on an `any` type.

Reply:
done